### ISSUE TITLE
Ensure compatibility with PHP 8

### ISSUE THREAD

**Issue by jeffpaul** [tlovett1/safe-redirect-manager#254]
**Describe the bug**
[Originally reported via WordPress.org](https://wordpress.org/support/topic/php-8-0-incompatibility-2/):

> I am on Php 8. When I activate the plugin it gives the error:

```
Deprecated: Required parameter $post follows optional parameter $actions in \wp-content\plugins\safe-redirect-manager\inc\classes\class-srm-post-type.php on line 98
```

> Is it possible for you to fix this so it can work good on Php 8?

**Steps to Reproduce**

**Expected behavior**
Ensure no errors/warnings thrown when running via PHP 8.

**Screenshots**

**Environment information**
 - Device: 
 - OS: 
 - Browser and version: 
 - WordPress version: 
 
 - Plugins and version: 
 - Theme and version: 
 
 - <details><summary>Site Health Info:</summary>
   
   </details>

**Additional context**

**Comment by skrlance** [comment]
Any help on this please!

**Comment by dkotter** [comment]
This was fixed in PR #231 so I think we're just waiting on a release here for that.
